src/frontend/src/lib/components/tokens/TokensZeroBalanceCheckbox.svelte

```diff
@@ -0,0 +1,18 @@
+<script lang="ts">
+	import { Checkbox } from '@dfinity/gix-components';
+	import { hideZeroBalances } from '$lib/derived/settings.derived';
+	import { i18n } from '$lib/stores/i18n.store';
+	import { hideZeroBalancesStore } from '$lib/stores/settings.store';
+
+	let checked: boolean;
+	$: checked = $hideZeroBalances;
```

Comment:
Should the value be assigned `onMount` only? Is `checked` triggered each time `toggleHide` is triggered so per extension click on the checkbox set checked twice to the same value?

Reply:
In my understanding, the flow is:

1. Click on checkbox
2. `toggleHide` triggered --> `hideZeroBalancesStore` is set to the opposite value
3. `hideZeroBalances` derived store is updated
4. `checked` is updated

No? I am missing something maybe...

I run it with `$: checked, console.log('checked', checked);` and it seems to update only once


https://github.com/user-attachments/assets/afd7ac3a-51a3-4977-b07a-09ba7bdaa3d4

Reply:
Yeah sorry, badly explained. It's what I meant.

1. Click on checkbox
2. toggleHide triggered --> hideZeroBalancesStore is set to the opposite value
3. hideZeroBalances derived store is updated
4. checked is updated // <---------------------------------- This rerender is not needed

But thinking about it, it's fine. If we use onMount, then in the future, if the state is used, there might be a discrepancy.

Nvm my comment, thanks for the feedback.